libminifi/include/controllers/ThreadManagementService.h

```diff
@@ -91,7 +91,7 @@ class ThreadManagementService : public core::controller::ControllerService {
   void yield() {
   }
 
-  bool isRunning() {
+  bool isRunning() const {
```

Comment:
I think this should be marked `override`, as well

Reply:
Updated in a20c3a130d80bd73e4383b8161e23c52c77fb1fd